specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,324 @@
+      pattern: ^[a-zA-Z0-9À-ÿ '\-\']{1,80}$
+      example: "Tulpstraat"
+    VertrokkenOnbekendWaarheen:
+      type: boolean
+      description: |
+        Indicatie dat de persoon is vertrokken naar het buitenland, maar dat niet bekend is waar naar toe.
+    Woonplaats:
+      description: |
+        Een woonplaats is een gedeelte van het grondgebied van de gemeente met een naam.
+      title: woonplaats naam
+      type: string
+      pattern: ^[a-zA-Z0-9À-ÿ '\-\']{1,80}$
+      example: "Duiven"
+    AbstractVerblijfplaatsBeperkt:
+      type: object
+      description: |
+        Gegevens over het verblijf of de woonlocatie van een persoon.
+      required:
+        - type
+      properties:
+        type:
+          type: string
+        functieAdres:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+        woonplaats:
+          $ref: '#/components/schemas/Woonplaats'
+        adresregel1:
+          $ref: '#/components/schemas/Adresregel1'
+        adresregel2:
+          $ref: '#/components/schemas/Adresregel2'
+        adresregel3:
+          $ref: '#/components/schemas/Adresregel3'
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaVerblijfplaats: '#/components/schemas/GbaVerblijfplaatsBeperkt'
+          Verblijfplaats: '#/components/schemas/VerblijfplaatsBeperkt'
+    VerblijfplaatsBeperkt:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaatsBeperkt'
+        - type: object
+          properties:
+            land:
+              $ref: 'land.yaml#/components/schemas/AbstractLand'
+            vertrokkenOnbekendWaarheen:
+              $ref: '#/components/schemas/VertrokkenOnbekendWaarheen'
+    GbaVerblijfplaatsBeperkt:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaatsBeperkt'
+        - type: object
+          properties:
+            straat:
+              $ref: '#/components/schemas/Straat'
+            huisnummer:
+              $ref: '#/components/schemas/Huisnummer'
+            huisletter:
+              $ref: '#/components/schemas/Huisletter'
+            huisnummertoevoeging:
+              $ref: '#/components/schemas/Huisnummertoevoeging'
+            aanduidingBijHuisnummer:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            postcode:
+              $ref: '#/components/schemas/Postcode'
+            locatiebeschrijving:
+              $ref: '#/components/schemas/Locatiebeschrijving'
+            land:
+              $ref: 'land.yaml#/components/schemas/GbaLand'
+    AbstractVerblijfplaats:
+      type: object
+      description: |
+        Gegevens over het verblijf of de woonlocatie van een persoon.
+      required:
+        - type
+      properties:
+        type:
+          type: string
+        functieAdres:
```

Comment:
functieAdres komt alleen voor bij een adres of lokatie, niet bij een verblijf buitenland